src/applications/vaos/components/TabNav.jsx

```diff
@@ -26,8 +26,26 @@ export function TabNav({ location, router }) {
           router.push('/');
           focusElement('#tabupcoming');
         }}
+        onNextTab={() => {
+          if (hasExpressCare) {
+            router.push('/express-care');
+            focusElement('#tabexpress-care');
+          }
+        }}
         title="Past appointments"
       />
+      {hasExpressCare && (
+        <TabItem
+          id="express-care"
+          tabpath="/express-care"
+          isActive={location.pathname === '/express-care'}
+          onPreviousTab={() => {
+            router.push('/past');
+            focusElement('#tabpast');
+          }}
+          title="Express Care"
```

Comment:
Has this been tested on mobile?  I think I recall Ryan mentioning we might want to shorten the tab titles if on mobile (e.g. Upcoming, Past, Express Care)

Reply:
Yeah, I need to check in about that, I thought they hadn't decided for sure yet.

Reply:
I got some updated guidance from Ryan. This PR is already huge, I'm going to leave it to a separate one.